app/assets/stylesheets/_settings.scss

```diff
@@ -104,29 +104,37 @@ $block-grid-max: 8;
 // 4. Base Typography
```

Comment:
As above comment, once you work with foundation a bit you will get used to the settings file. Once initial set up is done you don't really need to visit it too much.